README-CONAN.md

```diff
@@ -117,25 +202,208 @@ os_build=Windows
+| | arch_build=x86 | arch_build=x86 |
+
+### Modifying build settings in the Visual Studio IDE
+
+**CAUTION** You can change the runtime library in Visual Studio once you open the
+solution generated by CMake.  If you have generated a project to use the dynamic run-time library
+with the profile setting `compiler.runtime=MD` (or MDd), Visual Studio allows you to change that setting.
+However the libraries (provided by conan) to which you link are tied
+to a specific configuration and you will obtain linking errors.
+
+If you want to work with the static runtime library `MT` (or `MTd`), set up the profiles and build in a different directory such as build-static.
+
+### Changing profile settings with the conan command
+
+It is recommended that you create 8 profiles for msvc{2017|2015}{Release|Debug}{64|32}.
+
+You can modify profile settings on the command line.
+The following example demonstrates making substantial changes to profile settings by performing a 32 bit build using Visual Studio 2015 with a 2017 profile!
+
+```bash
+$ conan install .. --profile msvc2017Release64 -s arch_build=x86 -s arch=x86 -s compiler.version=14
+$ cmake         .. -G "Visual Studio 2015"
+$ cmake --build .  --config Release
+```
+
+### CMake/Cygwin Collisions on the build machine
+
+If you have Cygwin installed on your build machine, you may encounter the situation
+that CMake erroneously finds library files in Cygwin directories and adds `c:\\cygwin64\\usr\\include` to the
+compiler header search path.  FindIntl is a prime suspect and believe it's caused by %PATH%.
+
+If necessary, temporarily rename c:\\cygwin64\\usr\\include as c:\\cygwin64\\usr\\uncle to hide those files when working with CMake.
+
+<name id="2-3"></a>
+### 2.3) Cygwin Notes
+
+Do not use conan on the Cygwin Platform.  To build Exiv2 for Cygwin use CMake.  See [README-CMAKE](README-CMAKE)
+
+<name id="2-4"></a>
+### 2.4) MinGW Notes
+
+Team Exiv2 does not provide support for MinGW.
+
+Users have reported success with CMake (without conan) on both msys/1.0 and msys/2.0.
+
+[TOC](#TOC)
+
+<name id="3">
+## 3 Conan Architecture
+
+<name id="3-1">
+##### 3.1) conanfile.py
+
+In the root level of the **Exiv2** repository, the file `conanfile.py` defines C/C++ dependencies with the syntax: `Library/version@user/channel`
+
+For example, **zlib**:
+
+```python
+self.requires('self.requires('zlib/1.2.11@conan/stable')')
+```
+
+<name id="3-2">
+##### 3.2) Conan _**Recipes**_
+
+Conan searches remote servers for a _**recipe**_ to build a dependency.
+
+A _**recipe**_ is a python file which indicates how to build a library from sources. The recipe
+understands configurations: Platform/Compiler/Settings.  If the remote server has a pre-compiled package for
+your configuration, it will be downloaded.  Otherwise, conan will compile the libraries on your machine using instructions in the recipe.
+
+To illustrate, here is list of packages that returned by the command `$ conan search`
```

Comment:
My English is limited, but I think that we need to rephrase this line:

```
To illustrate, here is list of packages that returned by the command `$ conan search`
-->
To illustrate, here is the list of packages returned by the command `$ conan search`
```

Reply:
You are right.  This is an English grammar error.  It should say `... here is a list of ...`